src/app/components/shared/datetime/datetime/datetime.component.scss

```diff
@@ -0,0 +1,3 @@
+time {
+  border-bottom: 1px dotted;
+}
```

Comment:
so `<abbr>` uses dotted bottom border too.

Perhaps rather than a component specific style we want add some semantics to this. Something like `.has-context {}` in our shared styles section.